build_the_list/pdf_extract.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import print_function
+import argparse
+import os
+
+def extract_pdf_text(file_name):
+    tmp_filename = file_name + '.tmp'
+    os.system('pdf2txt.py {} > {}'.format(file_name, tmp_filename))
+    lines = []
+    with open(tmp_filename) as f:
+        [lines.append(line.strip()) for line in f if line != '\n']
+    os.remove(tmp_filename)
+    return lines
+
+def candidates(extracted_pdf):
```

Comment:
This will hopefully get abstracted once we do another one or two pdf text extractions.